### ISSUE TITLE
bug: AttributeError: 'BaseQuantizeConfig' object has no attribute 'get'

### ISSUE THREAD

**Issue by alexmukho** [bentoml/OpenLLM#301]
### Describe the bug

```openllm start llama --model-id TheBloke/Llama-2-7B-GPTQ --quantize gptq```

I'm getting this issue when trying to run quantized models: 'BaseQuantizeConfig' object has no attribute 'get'

Similar with `--quantize int8` option: AttributeError: 'BitsAndBytesConfig' object has no attribute 'get'

### Logs

```shell
2023-09-05T09:26:02+0000 [ERROR] [runner:llm-llama-runner:1] An exception occurred while instantiating runner 'llm-llama-runner', see details below:
2023-09-05T09:26:02+0000 [ERROR] [runner:llm-llama-runner:1] Traceback (most recent call last):
  File "/home/garret/.local/lib/python3.10/site-packages/bentoml/_internal/runner/runner.py", line 307, in init_local
    self._set_handle(LocalRunnerRef)
  File "/home/garret/.local/lib/python3.10/site-packages/bentoml/_internal/runner/runner.py", line 150, in _set_handle
    runner_handle = handle_class(self, *args, **kwargs)
  File "/home/garret/.local/lib/python3.10/site-packages/bentoml/_internal/runner/runner_handle/local.py", line 27, in __init__
    self._runnable = runner.runnable_class(**runner.runnable_init_params)  # type: ignore
  File "/home/garret/.local/lib/python3.10/site-packages/openllm/_llm.py", line 1143, in __init__
    if not self.model: raise RuntimeError('Failed to load the model correctly (See traceback above)')
  File "/home/garret/.local/lib/python3.10/site-packages/openllm/_llm.py", line 795, in model
    model = self.load_model(*self._model_decls, **self._model_attrs)
  File "/home/garret/.local/lib/python3.10/site-packages/openllm/_llm.py", line 265, in wrapper
    return f(self, *(*model_decls, *decls), **{**model_attrs, **attrs})
  File "/home/garret/.local/lib/python3.10/site-packages/openllm/serialisation/__init__.py", line 76, in caller
    return getattr(importlib.import_module(f'.{llm.runtime}', __name__), fn)(llm, *args, **kwargs)
  File "/home/garret/.local/lib/python3.10/site-packages/openllm/serialisation/transformers/__init__.py", line 177, in load_model
    model = infer_autoclass_from_llm(llm, config).from_pretrained(
  File "/home/garret/.local/lib/python3.10/site-packages/transformers/models/auto/auto_factory.py", line 516, in from_pretrained
    return model_class.from_pretrained(
  File "/home/garret/.local/lib/python3.10/site-packages/transformers/modeling_utils.py", line 2470, in from_pretrained
    quantization_method_from_config = config.quantization_config.get(
AttributeError: 'BaseQuantizeConfig' object has no attribute 'get'

2023-09-05T09:26:02+0000 [ERROR] [runner:llm-llama-runner:1] Traceback (most recent call last):
  File "/home/garret/.local/lib/python3.10/site-packages/starlette/routing.py", line 677, in lifespan
    async with self.lifespan_context(app) as maybe_state:
  File "/usr/lib/python3.10/contextlib.py", line 199, in __aenter__
    return await anext(self.gen)
  File "/home/garret/.local/lib/python3.10/site-packages/bentoml/_internal/server/base_app.py", line 75, in lifespan
    on_startup()
  File "/home/garret/.local/lib/python3.10/site-packages/bentoml/_internal/runner/runner.py", line 317, in init_local
    raise e
  File "/home/garret/.local/lib/python3.10/site-packages/bentoml/_internal/runner/runner.py", line 307, in init_local
    self._set_handle(LocalRunnerRef)
  File "/home/garret/.local/lib/python3.10/site-packages/bentoml/_internal/runner/runner.py", line 150, in _set_handle
    runner_handle = handle_class(self, *args, **kwargs)
  File "/home/garret/.local/lib/python3.10/site-packages/bentoml/_internal/runner/runner_handle/local.py", line 27, in __init__
    self._runnable = runner.runnable_class(**runner.runnable_init_params)  # type: ignore
  File "/home/garret/.local/lib/python3.10/site-packages/openllm/_llm.py", line 1143, in __init__
    if not self.model: raise RuntimeError('Failed to load the model correctly (See traceback above)')
  File "/home/garret/.local/lib/python3.10/site-packages/openllm/_llm.py", line 795, in model
    model = self.load_model(*self._model_decls, **self._model_attrs)
  File "/home/garret/.local/lib/python3.10/site-packages/openllm/_llm.py", line 265, in wrapper
    return f(self, *(*model_decls, *decls), **{**model_attrs, **attrs})
  File "/home/garret/.local/lib/python3.10/site-packages/openllm/serialisation/__init__.py", line 76, in caller
    return getattr(importlib.import_module(f'.{llm.runtime}', __name__), fn)(llm, *args, **kwargs)
  File "/home/garret/.local/lib/python3.10/site-packages/openllm/serialisation/transformers/__init__.py", line 177, in load_model
    model = infer_autoclass_from_llm(llm, config).from_pretrained(
  File "/home/garret/.local/lib/python3.10/site-packages/transformers/models/auto/auto_factory.py", line 516, in from_pretrained
    return model_class.from_pretrained(
  File "/home/garret/.local/lib/python3.10/site-packages/transformers/modeling_utils.py", line 2470, in from_pretrained
    quantization_method_from_config = config.quantization_config.get(
AttributeError: 'BaseQuantizeConfig' object has no attribute 'get'
```
```


### Environment

-`bentoml`: 1.1.5
- `transformers` version: 4.32.1
- Platform: Linux-5.15.0-82-generic-x86_64-with-glibc2.35
- Python version: 3.10.12
- Huggingface_hub version: 0.16.4
- Safetensors version: 0.3.3
- Accelerate version: 0.22.0
- Accelerate config: 	not found
- PyTorch version (GPU?): 2.0.1+cu117 (True)
- Tensorflow version (GPU?): not installed (NA)
- Flax version (CPU?/GPU?/TPU?): not installed (NA)
- Jax version: not installed
- JaxLib version: not installed
- Using GPU in script?: yes
- Using distributed or parallel set-up in script?: no

### System information (Optional)

Memory: 32Gb
CPU: Intel i7-10700KF
GPU: RTX 3080 10Gb

**Comment by aarnphm** [comment]
Hey can u try either 0.3.0?

**Comment by alexmukho** [comment]
@aarnphm I've tried 0.3.0 but unfortunately got "Segmentation fault" with any model. I have a fresh Ubuntu 22.04 with Cuda 11.8 and only OpenLLM with the required packages installed.

Tried to check with gdb:
```
(gdb) run openllm start opt
Starting program: /usr/bin/python3 openllm start opt
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".
[New Thread 0x7ffff4b7c640 (LWP 2045)]
[Thread 0x7ffff4b7c640 (LWP 2045) exited]
[New Thread 0x7ffff4b7c640 (LWP 2049)]
[New Thread 0x7fff3b780640 (LWP 2050)]
[New Thread 0x7fff38f7f640 (LWP 2051)]
[New Thread 0x7fff3877e640 (LWP 2052)]
[New Thread 0x7fff35f7d640 (LWP 2053)]
[New Thread 0x7fff3177c640 (LWP 2054)]
[New Thread 0x7fff2ef7b640 (LWP 2055)]
[New Thread 0x7fff2c77a640 (LWP 2056)]
[New Thread 0x7fff2bf79640 (LWP 2057)]
[New Thread 0x7fff27778640 (LWP 2058)]
[New Thread 0x7fff24f77640 (LWP 2059)]
[New Thread 0x7fff22776640 (LWP 2060)]
[New Thread 0x7fff1ff75640 (LWP 2061)]
[New Thread 0x7fff1f774640 (LWP 2062)]
[New Thread 0x7fff1af73640 (LWP 2063)]
[New Thread 0x7fff18772640 (LWP 2064)]

Thread 1 "python3" received signal SIGSEGV, Segmentation fault.
0x00005555556872e7 in _PyObject_GenericSetAttrWithDict ()```

**Comment by aarnphm** [comment]
oh yes can you try installing with `--no-binary :all:` for now? I'm fixing this issue atm

**Comment by alexmukho** [comment]
Great, now it starts.

But I'm getting the next error when running a query. Not sure if it's either OpenLLM related or not.
`RuntimeError: Expected all tensors to be on the same device, but found at least two devices, cuda:0 and cpu! (when checking argument for argument index in method wrapper_CUDA__index_select)`

```  File "/home/garret/.local/lib/python3.10/site-packages/bentoml/_internal/server/runner_app.py", line 271, in infer_single
    ret = await runner_method.async_run(*params.args, **params.kwargs)
  File "/home/garret/.local/lib/python3.10/site-packages/bentoml/_internal/runner/runner.py", line 55, in async_run
    return await self.runner._runner_handle.async_run_method(self, *args, **kwargs)
  File "/home/garret/.local/lib/python3.10/site-packages/bentoml/_internal/runner/runner_handle/local.py", line 62, in async_run_method
    return await anyio.to_thread.run_sync(
  File "/home/garret/.local/lib/python3.10/site-packages/anyio/to_thread.py", line 33, in run_sync
    return await get_async_backend().run_sync_in_worker_thread(
  File "/home/garret/.local/lib/python3.10/site-packages/anyio/_backends/_asyncio.py", line 2106, in run_sync_in_worker_thread
    return await future
  File "/home/garret/.local/lib/python3.10/site-packages/anyio/_backends/_asyncio.py", line 833, in run
    result = context.run(func, *args)
  File "/home/garret/.local/lib/python3.10/site-packages/bentoml/_internal/runner/runnable.py", line 143, in method
    return self.func(obj, *args, **kwargs)
  File "/home/garret/.local/lib/python3.10/site-packages/openllm/_llm.py", line 1185, in generate
    return self.generate(prompt, **attrs)
  File "/home/garret/.local/lib/python3.10/site-packages/openllm/_llm.py", line 943, in generate
    for it in self.generate_iterator(prompt, **attrs):
  File "/home/garret/.local/lib/python3.10/site-packages/openllm/_llm.py", line 981, in generate_iterator
    out = self.model(torch.as_tensor([input_ids]), use_cache=True)
  File "/home/garret/.local/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1501, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/garret/.local/lib/python3.10/site-packages/transformers/models/llama/modeling_llama.py", line 820, in forward
    outputs = self.model(
  File "/home/garret/.local/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1501, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/garret/.local/lib/python3.10/site-packages/transformers/models/llama/modeling_llama.py", line 662, in forward
    inputs_embeds = self.embed_tokens(input_ids)
  File "/home/garret/.local/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1501, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/garret/.local/lib/python3.10/site-packages/torch/nn/modules/sparse.py", line 162, in forward
    return F.embedding(
  File "/home/garret/.local/lib/python3.10/site-packages/torch/nn/functional.py", line 2210, in embedding
    return torch.embedding(weight, input, padding_idx, scale_grad_by_freq, sparse)```

**Comment by aarnphm** [comment]
This is fixed with 0.3.2 can you try again?

**Comment by alexmukho** [comment]
Works like a charm, thanks a lot.
